### PR TITLE
Updates for WCAG 2.2

### PR DESCRIPTION
**Pull request by iadawn**

### CODE REVIEW

**Comment by rvantonisse** [comment]
Looks good! Lot of folks are going to like this. 😄

**Comment by iadawn** [comment]
@shawna-slh Ready for publication

**Review by remibetin — CHANGES_REQUESTED** [review]

File: src/components/ui/Auditor/Criterion.svelte [hunk 1]
```diff
@@ -8,12 +8,12 @@
     <em class="criterion-header__level">Level {conformanceLevel}</em>
     <div class="criterion__resource-links">
       <ResourceLink
-        href="https://www.w3.org/WAI/WCAG21/Understanding/{id}.html"
+        href="https://www.w3.org/WAI/WCAG{wcagVersion}/Understanding/{id}.html"
```
[the remark this answers is not in the document]
Comment (remibetin) [reply src/components/ui/Auditor/Criterion.svelte:11]: @iadawn When I choose "WCAG 2.0", "Understanding 1.1.1" link points to https://www.w3.org/WAI/WCAG20/Understanding/text-equiv-all.html and does not work.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html is working though.

When I choose "WCAG 2.1", Understanding 1.1.1." link points to https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html and does work

[the remark this answers is not in the document]
Comment (shawna-slh) [reply src/components/ui/Auditor/Criterion.svelte:11]: @iadawn , those aren't the ones you were thinking of deleting though?

**Review by remibetin — CHANGES_REQUESTED** [review]

**Comment by remibetin** [comment]
@shawna-slh 

This PR and https://github.com/w3c/wai-wcag-em-report-tool/pull/127 can not be deployed together as they are.

To not break the French translation, new WCAG 2.2. SCs will need to be added to the French version of `wcag.json`.

If French translation is merged before WCAG 2.2.: to add in this PR.
If WCAG 2.2. is merged before French translation: to add in https://github.com/w3c/wai-wcag-em-report-tool/pull/127

**Comment by iadawn** [comment]
So, publish this PR so that 2.2 is supported in EM Report tool. Then update and publish #127 to provide the French translation?

**Comment by shawna-slh** [comment]
> To not break the French translation, new WCAG 2.2. SCs will need to be added (untranslated) to the French version of wcag.json.

What about not breaking other translations? Other one on our website is Nederlands. They are in same situation that 2.2 authorized translation is in progress.

Lower priority maybe we want to provide guidance to those who have forked and have localized version?

File: src/locales/nl/WCAG.json [hunk 2]
```diff
@@ -725,6 +725,51 @@
             "TITLE": "Statusberichten",
             "DESCRIPTION": "In content die is geïmplementeerd met opmaaktalen kunnen statusberichten door software bepaald worden met behulp van rol (role) of eigenschappen (properties), zodat hulptechnologieën de berichten aan de gebruiker kunnen presenteren zonder dat ze de focus krijgen.",
             "DETAILS": {}
+        },
```
[the remark this answers is not in the document]
Comment (remibetin) [reply src/locales/nl/WCAG.json:728]: I have opened an issue for this: https://github.com/w3c/wai-wcag-em-report-tool/issues/139

**Comment by shawna-slh** [comment]
Kevin OKed ready to publish (verbally in meeting now :)

**Pull request merged**